package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-nouislider",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
Bump the semver so people don't run into any problems (they shouldn't either way).